webview/src/shared/variables.scss

```diff
@@ -3,7 +3,7 @@ $bg-color: var(--vscode-editor-background);
 $bullet-size: 9px;
 $watermark-color: var(--vscode-descriptionForeground);
 
-$border-color: var(--checkbox-border);
+$border-color: var(--vscode-checkbox-border);
```

Comment:
There is at least one more of these in this file. `--button-primary-background`

Reply:
Weird, that variable actually doesn't have a `vscode` prefixed version 🤔